shogun-boot/src/main/resources/templates/index.html

```diff
@@ -192,6 +208,7 @@ <h3 class="pb-2 pt-4">Applications</h3>
           const html = `<shogun-app app='${app.id}'>` +
             `<a style='visibility: hidden' class='admin-btn' title='Edit application' slot='admin-btn' href='/admin/portal/application/${app.id}'>` +
             `<em class='fas fa-cog'></em></a>` +
+          // `<span slot='app-logo'>${app.}</span>` +
```

Comment:
I assume this line is currently missing the implementation of showing the specific logo of the application.